taccsite_cms/settings.py

```diff
@@ -404,6 +419,11 @@ def get_subdirs_as_module_names(path):
     576, 768, 992, 1200, 1400, 1680, 1920
 ]
 DJANGOCMS_PICTURE_RATIO = 1.618
+DJANGOCMS_PICTURE_ALIGN = [
+    ('left', _('Align left')),
+    ('right', _('Align right')),
+    ('center', _('Align center')),
+]
```

Comment:
**Note.** These are the [default values](https://github.com/django-cms/djangocms-picture/blob/master/djangocms_picture/models.py#L25-L30), but this offers a Python paper trail for devs to learn [how the `.article-left`|`right`|`center` classes are created](https://github.com/django-cms/djangocms-picture/blob/master/djangocms_picture/cms_plugins.py#L63), because those `.article-…` classes have **zero** default styles (thus no CSS paper trail).[^1]

[^1]: This means the alignment option—by default—is available to users but does not do anything! I made them functional by adding the styles in `taccsite_cms/static/site_cms/css/src/_imports/components/django.cms.picture.css`.